src/test/objectstore/BitAllocator_test.cc

```diff
@@ -449,7 +452,7 @@ TEST(BitAllocator, test_bmap_alloc)
   alloc->alloc_blocks(16, &start_block);
   delete alloc;
 
-  total_blocks = pow(BITMAP_SPAN_SIZE, 2) * 4;
+  total_blocks = pow(zone_size, 2) * 4;
   alloc_size = 16;
   alloc = new BitAllocator(total_blocks, zone_size, CONCURRENT, false);
   for (int64_t iter = 0; iter < 3; iter++) {
```

Comment:
if it is too much work, you can used your current approach of merging both. If you can get it work with both different that will be great!